pom.xml

```diff
@@ -145,8 +146,12 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
-
-
+            <!-- GCP sdk -->
+            <dependency>
+                <groupId>com.google.cloud</groupId>
+                <artifactId>google-cloud-storage</artifactId>
+                <version>${google-cloud-storage.version}</version>
```

Comment:
New dependency.

Reply:
Does this mean any users of SBAT will have to use GCP?

Reply:
This is only for openbanking to get the statement pdf file from gcp bucket instead of from the classpath to protect the statement of customer and avoid use the spring profiles to resolve the statement.

Reply:
Ah yes, I'm half awake! Sorry, ignore my comment!